docs/getting-started-guides/windows/index.md

```diff
@@ -1,139 +1,144 @@
+                               "Name":  "EndpointPolicy",
+                               "Value":  {
+                                             "Type":  "ROUTE",
+                                             "DestinationPrefix":  "11.0.0.0/8",
+                                             "NeedEncap":  true
+                                         }
+                           },
+                           {
+                               "Name":  "EndpointPolicy",
+                               "Value":  {
+                                             "Type":  "ROUTE",
+                                             "DestinationPrefix":  "10.127.132.213/32",
+                                             "NeedEncap":  true
+                                         }
+                           }
+                       ]
+}
 ```
 
 ## Starting the Cluster
-To start your cluster, you'll need to start both the Linux-based Kubernetes control plane, and the Windows Server-based Kubernetes node components. 
-## Starting the Linux-based Control Plane
-Use your preferred method to start Kubernetes cluster on Linux. Please note that Cluster CIDR might need to be updated.
-## Starting the Windows Node Components
-To start kubelet on your Windows node:
-Run the following in a PowerShell window. Be aware that if the node reboots or the process exits, you will have to rerun the commands below to restart the kubelet.
-
-1. Set environment variable *CONTAINER_NETWORK* value to the docker container network to use
-`$env:CONTAINER_NETWORK = "<docker network>"`
-
-2. Run *kubelet* executable using the below command
-`kubelet.exe --hostname-override=<ip address/hostname of the windows node>  --pod-infra-container-image="apprenda/pause" --resolv-conf="" --api_servers=<api server location>`
-
-To start kube-proxy on your Windows node:
+To start your cluster, you'll need to start both the Linux-based Kubernetes control plane, and the Windows Server-based Kubernetes node components (kubelet and kube-proxy). 
 
-Run the following in a PowerShell window with administrative privileges. Be aware that if the node reboots or the process exits, you will have to rerun the commands below to restart the kube-proxy.
-
-1. Set environment variable *INTERFACE_TO_ADD_SERVICE_IP* value to `vEthernet (KubeProxySwitch)` which we created in **_Windows Host Setup_** above
-`$env:INTERFACE_TO_ADD_SERVICE_IP = "vEthernet (KubeProxySwitch)"`
-
-2. Run *kube-proxy* executable using the below command
-`.\proxy.exe --v=3 --proxy-mode=userspace --hostname-override=<ip address/hostname of the windows node> --master=<api server location> --bind-address=<ip address of the windows node>`
+## Starting the Linux-based Control Plane
+Use your preferred method to setup and start Kubernetes cluster on Linux or follow the directions given in this [link](https://github.com/Microsoft/SDN/blob/master/Kubernetes/HOWTO-on-prem.md). Please note that Cluster CIDR might need to be updated.
```

Comment:
...since it's hard-coded to `11.0.0.0/16` in the scripts. Should we make that parameterized?

Also, though it's in the linked guide, reiterate the necessity of copying the certificate configuration file from the master.